Update PostHog vs Amplitude (#9278)
PostHog/posthog.com: contents/blog/posthog-vs-amplitude.mdx

---
title: 'In-depth: PostHog vs Amplitude'

> date: 2024-09-05

Author: ```suggestion
date: 2024-09-10
```

import { ComparisonHeader } from 'components/ComparisonTable/header'
import { ComparisonRow } from 'components/ComparisonTable/row'

## How is PostHog different?

### 1. Engineering and product-led

What does this mean?

- It means our pricing is 100% transparent. 
- It means engineers make product decisions.
- It means [we ship fast](/changelog) and iterate based on user feedback. 
- It means engineers do support – all product teams have a [weekly support hero](/handbook/engineering/support-hero).
- It means we work in the open – [our roadmap](/roadmap) and [company handbook](/handbook) are public.  
- It means you can use PostHog without ever jumping on a "quick call" – see [how we do sales](/sales).

> ### 2. Sustainably affordable

Reviewer: I dont really like the word "sustainably" here. Makes me think of envirionment/climate stuff. Can we use a different word here and in the paragraph? Maybe something along the lines of "scalable"

Author: Some options:

- Affordable, usage-based pricing
- Affordable and transparent pricing
- Fair, usage-based pricing
- Transparent, usage-based pricing

Author: I think the second one

Reviewer: I like the second one too

We're committed to [sustainably low pricing](/pricing/philosophy), and every customer gets a generous free monthly usage allowance. Inputting a card unlocks almost every feature, but you'll pay nothing if you stay within the free tier. See the [price comparison section](#price-comparison) for more.

### 3. Analytics and data warehouse in one

> Use our [built-in data warehouse](/data-warehouse) and SQL insight builder to analyze data from [Stripe](/tutorials/stripe-reports), [Hubspot](/tutorials/hubspot-reports), Salesforce, and [Zendesk](/tutorials/stripe-reports). Already have a warehouse? You can [link it with ours](/docs/data-warehouse/setup) to analyze your data in PostHog, and [batch export event and person data](/docs/cdp/batch-exports) from PostHog to your data warehouse.

Author: ```suggestion
Use our [built-in data warehouse](/data-warehouse) and SQL insight builder to analyze data from [Stripe](/tutorials/stripe-reports), [Hubspot](/tutorials/hubspot-reports), Salesforce, and [Zendesk](/tutorials/stripe-reports). 

Already have a warehouse? You can [link it with ours](/docs/data-warehouse/setup) to analyze your data in PostHog, and [batch export event and person data](/docs/cdp/batch-exports) from PostHog to your data warehouse.
```

import { ComparisonHeader } from 'components/ComparisonTable/header'
import { ComparisonRow } from 'components/ComparisonTable/row'

Amplitude and PostHog are superficially similar. 

They both offer product analytics and a range of additional tools, like feature flags, A/B testing, and session replay. 

> But, deep down, Amplitude is an analytics tool with some extra bits grafted on. In contrast, PostHog is built from the ground up as an all-in-one platform for building successful products. Here's how they compare.

Reviewer: INtro is too long IMO. Can we be more to the point?

Author: I could go back to just not having an intro again, tbh. It doesn't feel essential to have it.

## How is PostHog different?

### 1. We're engineering and product-led

What does this mean?

> - It means engineers make product decisions.
> - It means our pricing is 100% transparent.  
> - It means they do support – all product teams have a [weekly support hero](/handbook/engineering/support-hero).
> - It means [we ship fast](/changelog) and iterate based on user feedback. 
> - It means we work in the open – [our roadmap](/roadmap) is public, as is [our company handbook](/handbook). 
> - It means start using PostHog without ever jumping on a quick call – see [how we do sales](/sales).

Reviewer: Removed "they" as it was unclear who we're talking about. Also its confusing to have "they" and "we"
```suggestion
- It means engineers make product decisions.
- It means our pricing is 100% transparent.  
- It means engineers do support – all product teams have a [weekly support hero](/handbook/engineering/support-hero).
- It means [we ship fast](/changelog) and iterate based on user feedback. 
- It means we work in the open – [our roadmap](/roadmap) is public, as is [our company handbook](/handbook). 
- It means you can start using PostHog without ever jumping on a "quick call" – see [how we do sales](/sales).
```

### 2. We're building a genuine all-in-one platform

> We want you to spend less time engineering your data and more time engineering your product. You can ditch [Mixpanel](/blog/posthog-vs-mixpanel) and Amplitude for analytics, [Hotjar](/posthog-vs-hotjar) for replays and surveys, and [LaunchDarkly](/blog/posthog-vs-launchdarkly) for feature flags and experiments, with one seamlessly integrated platform. Need a data warehouse as well? We've [got you covered](/data-warehouse).

Reviewer: ON the surface Amplitude is also all-in-one. The paragraph didn't really tell me how we're different

### 3. We're cheaper and have a generous free tier

> We're committed to [sustainably cheap pricing](/blog/analytics-pricing) on all our products. Every user get a [generous free allowance](#free-tiers) every month. Inputting a card unlocks almost every feature, but you'll pay nothing if you stay within the free tiers. See the [price comparison section](#price-comparison) for more.

Reviewer: Maybe make this point #2 ? 

```suggestion
We're committed to [sustainably cheap pricing](/blog/analytics-pricing) on all our products. Every company gets a [generous free allowance](#free-tiers) every month. Inputting a card unlocks almost every feature, but you'll pay nothing if you stay within the free tiers. See the [price comparison section](#price-comparison) for more.
```

Author: Good idea.

  <ComparisonRow column1={true} column2={false} feature="Web analytics" description="Easy to use analytics for marketing websites" />
  <ComparisonRow column1={true} column2={true} feature="Session replay" description="Watch real users use your product, diagnose bugs" />
  <ComparisonRow column1={true} column2={true} feature="Feature flags" description="Roll out features safely, toggle features for cohorts or individuals" />
  <ComparisonRow column1={true} column2={true} feature="Experiments" description="Run tests on new features, optimize conversion funnels" />
  <ComparisonRow column1={true} column2={false} feature="User surveys" description="Collect and analyze feedback, run NPS and PMF surveys " />
  <ComparisonRow column1="Beta" column2={true} feature="Customer data platform" description="Sync data with third-party tools" />
  <ComparisonRow column1={true} column2={false} feature="Data warehouse" description="A single source for all your important data" />
</ComparisonTable>

### Product analytics

<ComparisonTable column1="PostHog" column2="Amplitude">

>   <ComparisonRow column1="1 million events" column2="50k tracked users (Starter plan only)" feature="Free usage" description="How much free usage do you get each month?" />

Reviewer: Can we avoid saying "free usage" twice?

>   <ComparisonRow column1="1 million events" column2="50k tracked users (Starter plan only)" feature="Free usage" description="How much free usage do you get each month?" />

Author: ```suggestion
  <ComparisonRow column1="1 million events" column2="50k tracked users (Starter plan only)" feature="Free usage" description="Free usage allowance each month?" />
```

  <ComparisonRow column1={true} column2={true} feature="Custom formulas" description="Use formulas to calculate unique insights" />
</ComparisonTable>

> > 💡 **Good to know:** PostHog supports [autocapture](/docs/product-analytics/autocapture), which means you can implement PostHog in mere minutes and ensure you don't miss out on events you haven't manually instrumented. Don't want autocapture? Just [turn it off](/docs/product-analytics/autocapture#disabling-autocapture) – we offer the best of both worlds.

Reviewer: I love all these callouts

  <ComparisonRow column1={true} column2={true} feature="Custom formulas" description="Use formulas to calculate unique insights" />
</ComparisonTable>

There are two key differences here:

> 1. PostHog supports autocapture, which makes it much faster to implement if you don't want to manually instrument every event. Don't want autocapture? Just [turn it off](/docs/product-analytics/autocapture) and use one of our SDKs – we offer the best of both worlds.
> 
> 2. PostHog has a [SQL-based query builder](/docs/hogql) for power users, which enables you to generate custom data analyses using PostHog event data. In future, our query builder will let you analyze data as you would in a business intelligence tool, like Metabase, inside PostHog.

Reviewer: should this be at the top before the table?
```suggestion
1. PostHog supports [autocapture](/docs/product-analytics/autocapture), which makes it much faster to implement if you don't want to manually instrument every event. Don't want autocapture? Just [turn it off](/docs/product-analytics/autocapture#disabling-autocapture) and use one of our SDKs – we offer the best of both worlds.

2. PostHog has a [SQL-based query builder](/docs/hogql) for power users, which enables you to generate custom data analyses using PostHog event data. In future, our query builder will let you analyze data as you would in a business intelligence tool, like Metabase, inside PostHog.
```

  <ComparisonRow column1={true} column2={false} feature="Early access" description="Manage betas, test features" />
</ComparisonTable>

- Feature flags make it easy to roll out features to specific users or groups, and [safely test in production](/product-engineers/testing-in-production). 

> - PostHog's feature flags are also tightly integrated with other features, so you can target session replays, surveys, and more using existing feature flags. See our guide on the [benefits of feature flags](/product-engineers/feature-flag-benefits-use-cases) for more.

Reviewer: Same comment. Feels a bit backwards to have this after the comparison table

  <ComparisonRow column1={true} column2={false} feature="Playlists" description="Sort recordings into static and dynamic playlists" />
  <ComparisonRow column1={true} column2={false} feature="Performance monitoring" description="Track network events within a session" />
  <ComparisonRow column1={true} column2={true} feature="Privacy masking" description="Censor personal information from playback" />
  <ComparisonRow column1={true} column2={false} feature="Conditional recording" description="Only capture the sessions you want" />
  <ComparisonRow column1={true} column2={false} feature="DOM explorer" description="Explore an interactive snapshot of replays" />
  <ComparisonRow column1={true} column2={false} feature="Export recordings" description="Save important recordings offline" />
  <ComparisonRow column1={true} column2={false} feature="Sample recorded sessions" description="Restrict the percentage of sessions that will be recorded" />
  <ComparisonRow column1={true} column2={false} feature="Record via feature flag" description="Only record sessions for users that have the flag enabled" />
</ComparisonTable>

> > 💡 **Good to know:** Replays let you watch how users experience your app, diagnose issues, improve support, and understand real user behavior in a way raw data can't. They are reconstructions of the session, _not_ video recordings of user's screen. Private info, such as passwords, is masked.

Reviewer: I think it should technically be "is" but I think "are" is better after reading it after "passwords"
```suggestion
> 💡 **Good to know:** Replays let you watch how users experience your app, diagnose issues, improve support, and understand real user behavior in a way raw data can't. They are reconstructions of the session, _not_ video recordings of user's screen. Private info, such as passwords, are masked.
```

| 25k                     | $249                 | $217        | <strong class="text-green">$1,128 per year cheaper</strong>  |
| 50k                     | $561                 | $430        | <strong class="text-green">$1,572 per year cheaper</strong>  |
| 75k                     | $841                 | $625        | <strong class="text-green">$2,592 per year cheaper</strong>  |
| 100k                    | $1,061               | $820        | <strong class="text-green">$2,892 per year cheaper</strong>  |
| 150k                    | $1,584               | $1,207      | <strong class="text-green">$4,524 per year cheaper</strong>  |
| 200k                    | $2,100               | $1,600      | <strong class="text-green">$6,000 per year cheaper</strong>  |
| 300k                    | $3,150               | $2,091      | <strong class="text-green">$12,708 per year cheaper</strong> |

We also provide numerous ways to [control how many events you send](/tutorials/fewer-unwanted-events) and reduce your costs, including the option to send cheaper **anonymous events**.

> ### Identified vs anonymous events

Reviewer: this section is way better now

Anonymous events **only include non-identifying data**, such as referral data (UTMs, domains, etc), device type, and basic location information. 

They're useful if you're tracking a huge number of users and only want to analyze aggregate data, such as on a content website with millions of visitors. 

> Anonymous events cheaper for us to ingest, so we charge you between 55% and 80% less for them. You'll only be charged the identified event rate when you call identifying functions like `posthog.identify()` or `posthog.group()`, or capture an event from an already-identified user. 

Reviewer: ```suggestion
Anonymous events are cheaper for us to ingest, so we charge you between 55% and 80% less for them. You'll only be charged the identified event rate when you call identifying functions like `posthog.identify()` or `posthog.group()`, or capture an event from an already-identified user. 
```

  <ComparisonRow column1={true} column2={false} feature="API mode" description="Create surveys via the API" />
</ComparisonTable>

- Surveys are great for tracking customer satisfaction scores like Net Promoter Score (NPS), customer satisfaction (CSAT), and customer effort score (CES).

- We include [survey templates](/templates?filter=type&value=survey) for all these and more helping you get started fast. Read our [guide comparing NPS, CSAT and CES](/product-engineers/nps-vs-csat-vs-ces) to see which one you should use and when.

## Price comparison

> ### Free tier

Reviewer: Hmm do we want to lean in very heavily here? Im not sure. I think we're starting to come off as a bit cheap if we focus on free tier too much. I worry this will scare big companies which expect to pay for prodcuts

Author: Fair. We cover the free usage in each product section, so we probably don't need to double up here.

   <ComparisonRow column1="Unlimited" column2="10 only" feature="Saved insights" description="Number of you charts you can create" />   
   <ComparisonRow column1={true} column2={false} feature="Custom events" description="Create custom tracking events" />
   <ComparisonRow column1={true} column2={false} feature="Cohorts" description="Combine users based on properties and events for group analysis" />
   <ComparisonRow column1={true} column2={false} feature="Formulas" description="Create custom insights using formulas" />
   <ComparisonRow column1={true} column2={true} feature="Feature flags" description="Release and control features" />
   <ComparisonRow column1={true} column2={false} feature="A/B testing" description="Run tests on users and groups" />
   <ComparisonRow column1={true} column2={false} feature="Surveys" description="Gather feedback and book user interviews" />
</ComparisonTable>

- Amplitude offers a free Starter plan, but it has many arbitrary limitations and a low limit on usage.

- Users who qualify for [our startup program](/startups) also get $50,000 in PostHog credit and a range of additional benefits. 

> ### Analytics pricing

Author: @raquelmsmith would like your input on this price comparison section just to make sure it's all correct. I'm adding more of these price comparison segments to emphasize our price advantage more.

> ### Analytics pricing

Reviewer: This section is very difficult to read tbh. Feels like an entire blog in itself. Can we simplify it?

The whole explaining anonymous vs identified events is too much. Its a difficult concept to wrap my head around

**PostHog** charges for usage – i.e. analytics events, recorded replays, feature flag API requests, and so on. No credit card is required to signup and use PostHog, but entering a credit card unlocks almost all features and **you'll pay nothing** if you stay within our **generous free tier**

> **Amplitude** charges based on monthly tracked users (MTUs). The cost of add owns, such as A/B testing (not available on self-serve plans), group analytics, and unlimited cohorts (self-serve plans are limited to five) isn't disclosed.

Reviewer 2: ```suggestion
**Amplitude** charges based on monthly tracked users (MTUs). The cost of add-ons, such as A/B testing (not available on self-serve plans), group analytics, and unlimited cohorts (self-serve plans are limited to five) isn't disclosed.
```

<ProductScreenshot
    imageLight = "https://res.cloudinary.com/dmukukwp6/image/upload/posthog_vs_amplitude_light_6accf68a1d.jpg" 
    imageDark = "https://res.cloudinary.com/dmukukwp6/image/upload/posthog_vs_amplitude_dark_6271c3bfa7.jpg"
    classes="rounded"
    alt="product analytics pricing"
/>

> PostHog is consistently cheaper for both product and web analytics. In the above comparison, we're assuming 75 identified events per monthly tracked user (MTU), which is the typical usage for most PostHog users.

Reviewer 2: I wonder if it's worth breaking this out for B2C vs B2B users. I _think_ B2C will have a much lower event-to-MTU ratio.

Author: I like this idea, but trying to avoid much more complexity atm.

| 5k                      | $124                 | $0          | <strong class="text-green">$1,488 per year cheaper</strong>  |
| 10k                     | $186                 | $0          | <strong class="text-green">$2,232 per year cheaper</strong>  |
| 25k                     | $186                 | $217        | <strong class="text-green">$1,128 per year cheaper</strong>  |
| 50k                     | $561                 | $430        | <strong class="text-green">$1,572 per year cheaper</strong>  |
| 75k                     | $841                 | $625        | <strong class="text-green">$2,592 per year cheaper</strong>  |
| 100k                    | $1,061               | $820        | <strong class="text-green">$2,892 per year cheaper</strong>  |
| 150k                    | $1,584               | $1,207      | <strong class="text-green">$4,524 per year cheaper</strong>  |
| 200k                    | $2,100               | $1,600      | <strong class="text-green">$6,000 per year cheaper</strong>  |
| 300k                    | $3,150               | $2,091      | <strong class="text-green">$12,708 per year cheaper</strong> |

> At each tier, PostHog is around 22% to 33% cheaper than Amplitude. Sending more than 75 identified events per MTU will increase the cost of PostHog relative to Amplitude, but we provide numerous ways to control how many events you send and recuce your costs, including the option to send cheaper **anonymous events**.

Reviewer 2: ```suggestion
At each tier, PostHog is around 22% to 33% cheaper than Amplitude. Sending more than 75 identified events per MTU will increase the cost of PostHog relative to Amplitude, and sending fewer events will reduce the cost. 

The benefit of charging per-event is that you can control how many events you send per user by only sending important events, whereas there is less desirable control for tracking fewer _users_. We provide [numerous ways](/tutorials/fewer-unwanted-events) to control how many events you send and reduce your costs, including the option to send cheaper **anonymous events**.
```

| <strong class="text-15px">50-100 million</strong>   | <span class="text-[15px] font-semibold">$0.0000364</span><span class="text-sm opacity-70">/event</span>  | <span class="text-[15px] font-semibold">$0.0000218</span><span class="text-sm opacity-70">/event</span> | <strong class="text-green">40% cheaper</strong> |
| <strong class="text-15px">100-250 million</strong>  | <span class="text-[15px] font-semibold">$0.0000187</span><span class="text-sm opacity-70">/event</span>  | <span class="text-[15px] font-semibold">$0.0000150</span><span class="text-sm opacity-70">/event</span> | <strong class="text-green">20% cheaper</strong> |
| <strong class="text-15px">250+ million</strong>     | <span class="text-[15px] font-semibold">$0.0000100</span><span class="text-sm opacity-70">/event</span>  | <span class="text-[15px] font-semibold">$0.0000090</span><span class="text-sm opacity-70">/event</span> | <strong class="text-green">10% cheaper</strong> |

> Anonymous events are the default event type. You'll only be charged the identified event rate when you call `posthog.identify`, or to capture an event from an identified user. 

Reviewer 2: ```suggestion
Anonymous events are the default event type. You'll only be charged the identified event rate when you call identifying functions like `posthog.identify()` or `posthog.group()`, or capture an event from an already-identified user. 
```

   <ComparisonRow column1="5,000 recordings" column2="1,000 recordings" feature="Free replay usage" description="Free tier usage each month" />
   <ComparisonRow column1="Unlimited" column2="10 only" feature="Saved insights" description="Number of you charts you can create" />   
   <ComparisonRow column1={true} column2={false} feature="Custom events" description="Create custom tracking events" />
   <ComparisonRow column1={true} column2={false} feature="Cohorts" description="Combine users based on properties and events for group analysis" />
   <ComparisonRow column1={true} column2={false} feature="Formulas" description="Create custom insights using formulas" />
   <ComparisonRow column1={true} column2={true} feature="Feature flags" description="Release and control features" />
   <ComparisonRow column1={true} column2={false} feature="A/B testing" description="Run tests on users and groups" />
   <ComparisonRow column1={true} column2={false} feature="Surveys" description="Gather feedback and book user interviews" />
</ComparisonTable>

Companies that qualify for [our startup program](/startups) also get $50,000 in PostHog credit and a range of additional benefits. 

> ### Can PostHog replace Google Analytics?

Reviewer: Should this be Amplitude?
